src/Web/Sfa.Das.Sas.ApplicationServices/Models/GroupedProviderSearchResults.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace Sfa.Das.Sas.ApplicationServices.Models
+{
+    public sealed class GroupedProviderSearchResults : ProviderSearchResults
```

Comment:
Should this be needed as the generic type could be used instead?

Reply:
The base types contains properties that are returned back to the UI. This was copying the same pattern that was already there. What I have done now is redefine this class not to use the base classes as they contained numerous properties that were not relevant in this scenario.